src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        Ball firstBall = Lottery.getRandomBall();
+        Ball secondBall = Lottery.getRandomBall();
+        Ball thirdBall = Lottery.getRandomBall();
+
+        System.out.println(firstBall);
+        System.out.println(secondBall);
+        System.out.println(thirdBall);
```

Comment:
can you somehow avoid code duplication here and make it shorter/simpler? Consider using loop for your purposes

Reply:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md read this at first

Reply:
not fixed